liveview-android/src/main/java/org/phoenixframework/liveview/data/dto/TextDTO.kt

```diff
@@ -31,6 +31,13 @@ import org.phoenixframework.liveview.ui.theme.textAlignFromString
 import org.phoenixframework.liveview.ui.theme.textDecorationFromString
 import org.phoenixframework.liveview.ui.theme.textStyleFromString
 
+/**
+ * High level element that displays text.
+ * ```
+ * <Text>My Text 1</Text>
+ * <Text text="My Text 2" />
+ * ```
+ */
```

Comment:
Out of curiosity, will these be used to generate documentation or is this just a comment?

Reply:
For now, my intention is to document for internal use, but as I am using the [KDoc](https://kotlinlang.org/docs/kotlin-doc.html) standard, documentation can be generated.